website/docs/patterns/filter-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,139 @@
+| [Radio Group](/components/form/radio) | Displays the values in a radio group, supporting the selection of a single value. |
+
+### Spacing
+
+Spacing within a filter sidebar extends the guidelines for spacing within a [form pattern](/patterns/form-patterns).
+
+- If a divider separates the groups use a `24px` vertical gap on the top and bottom of the divider.
+- If no divider is used, groups should receive a `32px` gap between them.
+
+**Ex 3.0: Spacing in a sidebar that uses dividers**
+
+![Sidebar spacing](/assets/patterns/filter-patterns/sidebar-spacing-divider.png =350x*)
+
+**Ex 3.1: Spacing in a sidebar that doesn't use dividers**
+
+![Sidebar spacing without a divider](/assets/patterns/filter-patterns/sidebar-spacing-no-divider.png =350x*)
+
+### Filters within a Flyout
+
+If its necessary for filters to overflow into a [Flyout](/components/flyout), use the same organizational method and spacing guidelines as the filter sidebar.
+
+![Filters within a Flyout](/assets/patterns/filter-patterns/flyout-filter-spacing.png)
+
+## Applied filters
+
+![Applied filters anatomy](/assets/patterns/filter-patterns/applied-filters-anatomy.png)
+
+| Element | Usage |
+|---------|-------|
+| Applied Filter Container | Wraps the label and applied filters to enforce consistent spacing |
+| Label | Labels the applied filters and communicates whether filters have been applied or not. |
+| Filters | A list of dismissible [Tags](/components/tag) that communicate what filters have been applied to the dataset. |
+| Bulk clear | Clear all applied filters with a single bulk action. Uses a small [Tertiary button](/components/button) with an icon. |
+
+### Spacing
+
+- In the applied filters container use a `16px` gap between elements.
+- Adhere to the spacing guidelines for the [Tag](/components/tag#spacing) between multiple filters.
+
+![Applied filters spacing](/assets/patterns/filter-patterns/applied-filters-spacing.png)
+
+### Empty state
+
+If no filters are applied, indicate this to the user by changing the text in the label and optionally providing details on filtering via a [Tooltip](/components/tooltip).
+
+![Applied filters empty state](/assets/patterns/filter-patterns/applied-filters-empty-state.png)
+
+### Layout
+
+Depending on the available space, the size of the viewport, and the number filters that have been applied, the list of applied filters may overflow to multiple lines. In this circumstance it's recommend to stack the label, filter list, and bulk clear action in a column orientation.
+
+![Applied filters multi-line](/assets/patterns/filter-patterns/applied-filters-overflow-layout.png)
+
+Use a `12px` gap between each row in this orientation.
+
+![Applied filters multi-line spacing](/assets/patterns/filter-patterns/applied-filters-overflow-layout-spacing.png)
+
+## Pagination
+
+Refer to the [Pagination](/components/pagination) guidelines for more information on how to use Pagination in relation to a data set.
```

Comment:
should this paragraph refer to the `Pagination` _specifications_, and not the _guidelines_?

Reply:
Yep, that's a good point, I'll update!